HTMLPrettify.py

```diff
@@ -185,13 +188,34 @@ def exists_in_path(cmd):
     return False
 
   @staticmethod
+  def ask_to_set_node_path():
+    # Usually, it's just node.js not being found. Try to alleviate the issue.
+    msg = "Node.js was not found in the default path. Please specify the location."
+    if not sublime.ok_cancel_dialog(msg):
+      msg = "You won't be able to use this plugin without specifying the path to node.js."
+      sublime.error_message(msg)
+    else:
+      PluginUtils.open_sublime_settings(self.view.window())
+
+  @staticmethod
   def get_node_path():
     platform = sublime.platform()
     node = PluginUtils.get_pref("node_path").get(platform)
+    if not os.path.exists(node):
+      return False
+      raise IOError
     print("Using node.js path on '" + platform + "': " + node)
     return node
 
   @staticmethod
+  def get_node_or_node_path():
+    if PluginUtils.exists_in_path("node"):
+      print("Using node.js in path")
+      return "node"
```

Comment:
We've had issues in the past where even though `exists_in_path("node")` would return true, invoking `run.js` just with `node` and not a full path would still not work. Maddening. Don't know if Sublime fixed the paths issues in the meantime, but it's dangerous to revert to this now.

I like everything about this patch except using `get_node_or_node_path`. Feel free to use it in your own branch though and I'd love to find out if there's no issues.